test/fixtures/tasks.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  description: MyString
+  complete: false
+
+two:
+  description: MyString
+  complete: false
```

Comment:
You could also delete these unused fixtures. We typically use FactoryBot, but that won't be that helpful until you write some tests. I think upcase will be a good start and you can always ping me or ask n #dev if you have questions